_posts/2021-02-01-Using-Tables-In-My-Website.md

```diff
@@ -0,0 +1,8 @@
+---
+layout: post
+title: Using Tables In My Tables
+description: 
+summary: 
+tags:
+---
+This week I will be using my chromebook beacause I haven't gotten my Macbook. In the morning I worked on typing club for about 35 minutes.Then I logged into my email and onto slack. Then Kajari helped me with my website, we worked on adding tables to the webiste to make it flow. Every time I wanted to make a change on my website I had to commit it locally because I do not have the Macbook which would have made it a little easier to see my progress faster. One change I hope to make is to minimize the ammount of white space by making an image more lenghty.
```

Comment:
`commit it locally` --  locally would be if you could make the changes on your computer. Instead what you were doing was you were committing to the main branch directly on github, and then github was building the website for you fully. So change the text here: `commit it locally` to reflect that.

Also, the reason you had to commit it directly on github is not because you didn't have your Macbook, but because your Chromebook doesn't have a text editor and won't let you download one because of the school's security permissions.

ammount is spelled `amount`

lenghty is spelled `lengthy`